src/app/file-browser/file-list/file-list.component.ts

```diff
@@ -310,20 +313,12 @@ export class TerraFileListComponent implements OnInit, AfterViewInit, OnChanges,
     {
         if(!isNullOrUndefined(this.activeStorageService))
         {
-            if(this.activeStorageService.isPublic)
+            if(this.activeStorageService instanceof TerraBasePrivateStorageService)
             {
```

Comment:
Was ist die Intension dieses IF's?
Mir wird hier nicht klar für was die jeweilige Struktur unten stehen soll.
Vllt. auslagern in jeweils eine Funktion? 
Oder haben beide den gleichen Hintergrund  und  nur sollen anders aussehen? Wenn ja würde ich nur diesen Unterschied über ein IF Regeln. Ich gehe aber mal vom ersten aus.